### DISCUSSION TITLE
How to Assign eSIM to RM5XX module?

### DISCUSSION THREAD

**Discussion by droideggs** [iamromulan/cellular-modem-wiki#81] [Help - Quectel]
I would like to assign an eSIM to the RM502Q-AE module. I took at look at the eSIM LPA documentation and it seems as though many of the AT commands give an error. What steps are needed to properly assign an eSIM to the module? Thanks in advance

**Comment by iamromulan** [comment]
I haven't messed with this yet but from what I heard there's some Quectel Linux utility called IPAD that assists with this. 

Last I remember you could find it in this treasure chest:
https://cnquectel-my.sharepoint.com/:f:/g/personal/america-fae_quectel_com/Eojcy41tWUFOnUwLTDRIREgB_sxQXcjY6CJIjJ9mTrZ35g?e=y89kBf
